### DISCUSSION TITLE
How to Turn Off Blood and Gore

### DISCUSSION THREAD

**Discussion by zicklag** [redeclipse/base#1223] [Help and Support] [answered]
Hey there, I was just wondering how to turn off blood and gore in Red Eclipse 2 and it took me a while to figure it out after having to grep on the codebase so I figured I'd post this discussion for anybody who ends up searching the repo for the same thing. So, to answer my own question:

1. Start a match
2. Press the back-tick ( ` ) button to open the console
3. Type `nogore 2` and press enter

Voila!

**Comment by zicklag** [comment, marked as answer]
Answered in original post. :smile:

**Comment by meamdumy** [comment]
Thank you so much. My parents are sticklers for anything with blood, and now I don't have to worry about getting caught and having to go back to tesseract
